### ISSUE TITLE
Can't connect to server

### ISSUE THREAD

**Issue by holorum** [FLOODY88/ButlerSpeak#1]
I've tried both 127.0.0.1 and the server's IP; even the domain.
The login name, and the password are correct, copied it from the configuration file to YaTQA, it works from my computer.
I try to run the bot on Ubuntu, with 11.0.2 version of openjdk.
It gives back:
`user@domain:/path/to/bot$ java -jar ./butlerspeak-1.0.jar 
Exception in thread "main" com.github.theholywaffle.teamspeak3.api.exception.TS3ConnectionFailedException: Could not connect to the TeamSpeak3 server
	at com.github.theholywaffle.teamspeak3.QueryIO.<init>(QueryIO.java:84)
	at com.github.theholywaffle.teamspeak3.TS3Query.connect(TS3Query.java:128)
	at me.floody.butlerspeak.ButlerSpeak.<init>(ButlerSpeak.java:56)
	at me.floody.butlerspeak.ButlerSpeak.main(ButlerSpeak.java:36)
Caused by: java.net.SocketTimeoutException: Read timed out
	at java.base/java.net.SocketInputStream.socketRead0(Native Method)
	at java.base/java.net.SocketInputStream.socketRead(SocketInputStream.java:115)
	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:168)
	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:140)
	at java.base/sun.nio.cs.StreamDecoder.readBytes(StreamDecoder.java:284)
	at java.base/sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:326)
	at java.base/sun.nio.cs.StreamDecoder.read(StreamDecoder.java:178)
	at java.base/java.io.InputStreamReader.read(InputStreamReader.java:185)
	at java.base/java.io.BufferedReader.fill(BufferedReader.java:161)
	at java.base/java.io.BufferedReader.readLine(BufferedReader.java:326)
	at java.base/java.io.BufferedReader.readLine(BufferedReader.java:392)
	at com.github.theholywaffle.teamspeak3.SocketReader.<init>(SocketReader.java:70)
	at com.github.theholywaffle.teamspeak3.QueryIO.<init>(QueryIO.java:73)
	... 3 more
`

I'd like to know what causes the problem and/or how can I fix it, and I would thank it.

**Comment by FLOODY88** [comment]
Hi @holorum, could you please provide the output of `java -version`. Please also ensure that `server.host` only contains the hostname (note that the port is **not** part of the hostname), so e.g. `localhost`.

Do you use SSH as server protocol or did you change the query port?

**Comment by holorum** [comment]
the java's version is:
`$` java -version
openjdk version "11.0.2" 2019-01-15
OpenJDK Runtime Environment (build 11.0.2+9-Ubuntu-3ubuntu118.04.3)
OpenJDK 64-Bit Server VM (build 11.0.2+9-Ubuntu-3ubuntu118.04.3, mixed mode, sharing)`
I did not add the port to the hostname; I noticed how it works, and gave the server's port, and the query's port each different.
I did not use SSH, but changed the port; after a reconfiguration and a system restart, I noticed that I had changed the port, but forgot to change it back, because at first it wouldn't log me in, so I thought "Okay, so this must be using SSH" for whatever no reason. But I really just misspelled the login name, and corrected it.
Thank you still for your answer.

tl;dr: I had changed the query port, but forgot to change it back, and that was the problem.
